### ISSUE TITLE
Try-Out Servers List is Empty

### ISSUE THREAD

**Issue by ralusnom** [api-platform/api-platform#1611]
When I try to ***Try it out*** I cannot press the execute button. 
I see this error in the console-tab as soon as I press that button.

![image](https://user-images.githubusercontent.com/266216/89335010-b7c88900-d697-11ea-99e8-c074444aa6a2.png)

Is there something I forgot to configure? It is still the default recipe configuration, the only thing I changed is the namespace of the entities, since they are not in src/Entity.

I'm using "api-platform/core": "^2.1" on symfony 5.2

**Comment by cnizzardini** [comment]
We are also experiencing this same issue.

**Comment by cnizzardini** [comment]
p.s. i added this to my `composer.json` and it resolved the issue after running a `composer install`:

https://getcomposer.org/doc/04-schema.md#prefer-stable

There is something yucky in dev so prefer stable.

**Comment by malaimo2900** [comment]
Yes, using @cnizzardini solution is working.

I wanted to add you can also choose a minimum-stability: stable.  This also corrects the issue.

You may want to try removing the api package, deleting the vendors/* directories and then adding api module again with composer before running composer install.

**Comment by ralusnom** [comment]
Yes, it works now after setting ` "prefer-stable": true,` thanks @cnizzardini!

**Comment by jordan15** [comment]
For me this solution does not working, what is the package in cause of this issue ?

**Comment by soyuka** [comment]
Closing per #2987. Questions are best asked on [GitHub Discussions](https://github.com/api-platform/core/discussions) or [Stack Overflow (api-platform tag)](https://stackoverflow.com/questions/tagged/api-platform.com). The framework now lives at `api-platform/core`.
